system-test/pubsub.ts

```diff
@@ -862,6 +862,30 @@ describe('pubsub', () => {
           subscription.close(done);
         });
       });
+
+      it('should seek to a future date (purge)', done => {
+        const testText = 'Oh no!';
+
+        // Forward-seek to remove any messages from the queue (those were
+        // placed there in before()).
+        //
+        // We... probably won't be using this in 3000?
```

Comment:
I look forward to someone at a hover desk addressing this bug.

Reply:
<Marty McFly meme>